experiment/autobumper/bumper/bumper.go

```diff
@@ -39,13 +40,33 @@ const (
 	testImageRepo   = prowRepo
 )
 
-func Call(cmd string, args ...string) error {
+func Call(stdout, stderr io.Writer, cmd string, args ...string) error {
 	c := exec.Command(cmd, args...)
-	c.Stdout = os.Stdout
-	c.Stderr = os.Stderr
+	c.Stdout = stdout
+	c.Stderr = stderr
 	return c.Run()
 }
 
+type SecretCensor interface {
+	Censor(content []byte) []byte
+}
+
+type HideSecretsWriter struct {
+	Delegate io.Writer
+	SC       SecretCensor
```

Comment:
nit: `SC` is an inscrutable name, maybe use `censor`

Reply:
Done.